modules/porous_flow/Makefile

```diff
@@ -17,9 +17,10 @@ FRAMEWORK_DIR      ?= $(MOOSE_DIR)/framework
 include $(FRAMEWORK_DIR)/build.mk
 include $(FRAMEWORK_DIR)/moose.mk
 
-# dependency on tensor_mechanics and fluid_properties
+# dependency on tensor_mechanics and fluid_properties and chemical_reactions
 TENSOR_MECHANICS := yes
 FLUID_PROPERTIES := yes
+CHEMICAL_REACTIONS := yes
```

Comment:
Whoa! This will require a change to modules.mk too. This is for the case where somebody uses only PourousFlow from yet another application (Maybe Ba-hun) but doesn't specify "chemical reactions".